hexrd/ui/interactive_template.py

```diff
@@ -41,20 +74,17 @@ def rotate_shape(self, angle):
         self.rotate_template(self.shape.xy, angle)
         self.redraw()
 
-    def create_shape(self, module, file_name, det, instr):
+    def create_polygon(self, verts, **kwargs):
```

Comment:
Can you rename `kwargs` to `polygon_kwargs` for clarity?

Reply:
Sure thing, done.